typo3Model.xml

```diff
@@ -41,6 +41,27 @@
       <type name="dkd:typo3:pages">
           <title>TYPO3 Page</title>
           <parent>cm:folder</parent>
+          <properties>
+              <property name="dkd:typo3:pages:doktype">
+                  <type>d:int</type>
+                  <mandatory>true</mandatory>
+                  <multiple>false</multiple>
+              </property>
+          </properties>
+          <associations>
+              <association name="dkd:typo3:pages:shortcut_to">
```

Comment:
> In cmis there is and so there must be also a way in alfresco.

That's how I came up with the association. As far as I understand this does get added as a relationship type (using the specified "name" as id), in this case one that is more specific than a plain "references" relationship in that it restricts targets to only be TYPO3 pages. My Q was more along the lines of, did I miss a built-in type that's an obvious candidate but not generic like "references".